File: app/src/main/java/com/gongbo/fss/demo/adapter/ListDataModel.java
package com.gongbo.fss.demo.adapter;

import java.util.ArrayList;
import java.util.List;

public class ListDataModel {
    public static List<String> getDatas() {
        List<String> datas = new ArrayList<>();
        for (int i = 0; i < 50; i++) {
            datas.add(i + "");
        }
        return datas;
    }
}

File: app/src/main/java/com/gongbo/fss/demo/adapter/ListDataModel.kt
package com.gongbo.fss.demo.adapter

import java.util.ArrayList

object ListDataModel {
    val datas: List<String>
        get() {
            val datas = ArrayList<String>()
            for (i in 0..49) {
                datas.add(i.toString() + "")
            }
            return datas
        }
}
